src/rev_ai/models/asynchronous/job.py

```diff
@@ -28,7 +29,9 @@ def __init__(
             rush=None,
             segments_to_transcribe=None,
             remove_atmospherics=None,
-            speakers_count=None):
+            speakers_count=None,
+            summarization: Summarization = None,
+            translation: Translation = None):
```

Comment:
```suggestion
            speakers_count=None,
            summarization: Summarization=None,
            translation: Translation=None):
```

Reply:
Same as bellow:

```
I follow PEP8, which is commonly accepted standard for python formatting, PyCharm supports PEP 8 and suggests this formatting here. I Used autoformat everywhere where reasonable.

Reply:
Discussed on Zoom. Agreed that formatting of arguments with type annotation is different according to the standard.